index.html

```diff
@@ -0,0 +1,94 @@
+
+<!doctype html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+	<link  rel="stylesheet" type="text/css" href="style.css">
+	<title>Cubes</title>
+</head>
+<body>
+
+	<section class="wrapper">
+	
+		<div class="weather-widget">
+
+			<div class="weather-widget__card weather-widget__card_blue">
+				<img src="img\chicago.png" alt="chicago" />
+				<div class="widget__card_info">Chicago / 23°</div>

```

Comment:
When using BEM methodology, for small blocks, we may use only one block as a "base", and name everything inside it like "block__element". But, in the case of this weather widget, its structure is quite complex. So, in order to clearly organize our CSS classes, we may use the "1st level" elements as "2nd level" blocks for elements inside them. 

For example:
```
<div class="weather-widget">
    <div class="weather-widget__card weather-widget__card_blue">
        <div class="card__info">Chicago / 23°</div>
    </div>
</div>
```
Here, `blue` is a modifier, as we have lots of `weather-widget__card` which are the same and differ only in color - hence, the color modifier. 
On the other hand, `info` is not a modifier, as it is used inside each and every `weather-widget__card`. It is an element of the `card`.

In this course, the next homework will not be related to this widget, so I write this as advice for your future projects - you are not obligated to do these changes.

Good luck with your next homework!